Marsiple/App/AppDelegate.swift

```diff
@@ -10,10 +10,12 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
     var window: UIWindow?
```

Comment:
Btw, we can initialise the window immediately within the variable definition area, making it a `let non-optional`, instead of a `var optional`.

`var window: UIWindow?` -> `let window = UIWindow(frame: UIScreen.main.bounds)`